packages/flutter/test/material/floating_action_button_location_test.dart

```diff
@@ -684,7 +684,7 @@ void main() {
       const double keyboardHeight = 200.0;
       const double viewPadding = 50.0;
       final Key floatingActionButton = UniqueKey();
-      const double bottomNavHeight = 99.0;
+      const double bottomNavHeight = 106.0;
```

Comment:
Why does this test need to be changed as a result of this PR?

Reply:
This test sets the view padding as well as the keyboard insets, so although this change isn't related to the keyboard being open or closed, the bottom nav height still needed to be updated.